hack/containerd.toml

```diff
@@ -1 +1,2 @@
-subreaper = true
```

Comment:
why remove `subreaper=true`?

Reply:
We have been running without it for a long time now, which shows that our reaper in `machined` is working. This config was never really used. We were copying it into the wrong location.